misk-launchdarkly-core/src/main/kotlin/misk/feature/launchdarkly/LaunchDarklyFeatureFlags.kt

```diff
@@ -132,13 +142,22 @@ class LaunchDarklyFeatureFlags @Inject constructor(
       }
     }
     if (attributes.number != null) {
-      attributes.number!!.forEach { (k, v) -> builder.privateCustom(k, v) }
-    }
-    if (attributes.anonymous) {
-      // This prevents the user from being stored in the LaunchDarkly dashboard, see
-      // https://docs.launchdarkly.com/docs/anonymous-users
-      builder.anonymous(true)
+      attributes.number!!.forEach { (k, v) ->
+        if (v !is Int || v !is Double) {
+          logger.info { "Deprecated attribute type, please use an Int or Double for $k" }
```

Comment:
I should really split out misk logging, but for now can you use kotlin-logging directly so we're not pulling in all of core?